Userland/Libraries/LibJS/Runtime/GlobalObject.h

```diff
@@ -81,6 +81,9 @@ class GlobalObject : public ScopeObject {
     // Not included in JS_ENUMERATE_NATIVE_OBJECTS due to missing distinct prototype
     ProxyConstructor* m_proxy_constructor { nullptr };
 
+    // Not included in JS_ENUMERATE_NATIVE_OBJECTS due to missing distinct constructor
+    GeneratorObjectPrototype* m_generator_object_prototype { nullptr };
```

Comment:
```suggestion
    // Not included in JS_ENUMERATE_NATIVE_OBJECTS due to missing distinct prototype
    ProxyConstructor* m_proxy_constructor { nullptr };
    GeneratorObjectPrototype* m_generator_object_prototype { nullptr };
```

Reply:
Well this wouldn't be correct anymore. The proxy constructor doesn't have a companion prototype, but the generator object prototype doesn't have a companion constructor. They're both outliers, but for slightly different reasons.

Reply:
oh, I see it now. Nevermind :sweat_smile: